src/assets/src/components/queueManager.tsx

```diff
@@ -0,0 +1,446 @@
+import * as React from "react";
+import { useState, createRef, ChangeEvent } from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from "@fortawesome/free-solid-svg-icons";
+import { Alert, Button, Col, Form, InputGroup, Modal, Row, Table } from "react-bootstrap";
+import Dialog from "react-bootstrap-dialog";
+import 'react-phone-input-2/lib/style.css';
```

Comment:
I changed the name of `edit.tsx` to `queueManager.tsx`, since that made more sense to me. Apparently that and all the changes I made was enough for GitHub to think it was another file (you'll see the deleted `edit.tsx` above).